vendor/modules.txt

```diff
@@ -9,6 +9,9 @@ github.com/apparentlymart/go-textseg/v13/textseg
 github.com/fatih/color
 # github.com/go-test/deep v1.0.6
 ## explicit; go 1.13
+# github.com/gofrs/uuid v4.4.0+incompatible
```

Comment:
Any reason for using this lib ?

Google/uuid package is already in this go.mod and having 2 dependencies on different uuid pkgs isnt good

And the reason I prefer keeping `google/uuid` over refactoring to only use the new one is because any dev which just quickly wants an UUID V4 would default to `google/uuid` without thinking that we're actually using `gofrs/uuid`

Reply:
I looked over docs and seems like `google/uuid` cant generate V5 UUIDs.

In this case I'm okay with having two different libs for UUID handling, but we should get rid of `google/uuid` as soon as possible and replace all those calls with `gofrs/uuid` equivalent

Reply:
@avirtopeanu-ionos ` but we should get rid of google/uuid as soon as possible and replace all those calls with gofrs/uuid equivalent` I agree, I'll open an issue for this.